src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.Accessor.cs

```diff
@@ -57,7 +57,7 @@ public Accessor(SQLitePersistentStorage storage)
             public async Task<Checksum> ReadChecksumAsync(TKey key, CancellationToken cancellationToken)
             {
                 using (var stream = await ReadBlobColumnAsync(key, ChecksumColumnName, checksumOpt: null, cancellationToken).ConfigureAwait(false))
-                using (var reader = ObjectReader.TryGetReader(stream, cancellationToken))
+                using (var reader = ObjectReader.TryGetReader(stream, leaveOpen: false, cancellationToken))
```

Comment:
💡 No need to leave this open since the containing using statement will already dispose of `stream`